### PR TITLE
AWS CLI: Fix algorithm to find the latest version

### PR DESCRIPTION
**Pull request by g3rhard**
Hello, just found that I did a mistake in algorithm - it started always showing the wrong version, instead of latest one:

```
> Get-EvergreenApp -Name "AWSCLI"

Version Type URI                                            
------- ---- ---                                            
2.9.9   msi  https://awscli.amazonaws.com/AWSCLIV2-2.9.9.msi
```

After the changes it should be fine:

```
> Get-EvergreenApp -Name "AWSCLI"

Version Type URI                                             
------- ---- ---                                             
2.9.23  msi  https://awscli.amazonaws.com/AWSCLIV2-2.9.23.msi
```

### CODE REVIEW

File: Evergreen/Apps/Get-AWSCLI.ps1 [hunk 1]
```diff
@@ -21,14 +21,16 @@
         ContentType  = $res.Get.Update.ContentType
         ReturnObject = "Content"
     }
-    $Content = Invoke-WebRequestWrapper @params | ConvertFrom-Json
+
+    # Get only latest version tag from GitHub API
+    $Content = ((Invoke-WebRequestWrapper @params | ConvertFrom-Json).name | %{ new-object System.Version ($_) } | Sort-Object -Descending | Select-Object -First 1 | % {("{0}.{1}.{2}" -f $_.Major,$_.Minor,$_.Build)})
```
[the remark this answers is not in the document]
Comment (aaronparker) [reply Evergreen/Apps/Get-AWSCLI.ps1:26]: @g3rhard could you address this issue please? Also update `new-object System.Version` with `New-Object -TypeName "System.Version"`

[the remark this answers is not in the document]
Comment (aaronparker) [reply Evergreen/Apps/Get-AWSCLI.ps1:26]: Please fix this one too

**Review by aaronparker — CHANGES_REQUESTED** [review]
Excellent. A couple of code items to address.

I need too add a contributing guide, but essentially I want to avoid aliases in the code.

**Review by aaronparker — APPROVED** [review]

**Comment by g3rhard** [comment]
> Excellent. A couple of code items to address.
> 
> I need too add a contributing guide, but essentially I want to avoid aliases in the code.

Okay, not a problem at all. we all fighting for the clean code :)

**Comment by aaronparker** [comment]
Merging changes and bypassing require signed commit check

**Pull request merged**